mindsdb/integrations/handlers/writer_handler/requirements.txt

```diff
@@ -0,0 +1,13 @@
+langchain
+pygpt4all==1.1.0
+chromadb
+urllib3==1.26.6
+numpy
+pandas
+torch>=1.13.0,<1.14.0
```

Comment:
Why 1.13, specifically? Ideally we could use >= 2.0 here, otherwise there will be a conflict with Lightwood

Reply:
I was having some issues getting the deps to resolve and that range seemed to fix it on my side. I will be tweaking the pins this am as Tom said he wasn't able to install

Reply:
In fact, I can see it was specifically lightwood which was saying that it needs to torch>=1.13.0,<1.14.0

Reply:
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
lightwood 23.4.3.0 requires torch<1.14.0,>=1.13.0, but you have torch 2.0.1 which is incompatible.